main.py

```diff
@@ -78,5 +80,67 @@ def calorie_consumption_calculator(food_type: str, food_mass: int, food_data: di
     return quantity_of_eaten_calories
 
 
+def aerobic_exercise_calories_burning(user_weight: int, exercise_name: str, exercise_time: float,
+                                      exercise_intensity: int = 3) -> float:
+    """
+    Function to calculate burned calories after aerobic type of exercise
+    :param exercise_name: name of exercise (str)
+    :param exercise_time: exercise duration (float)
+    :param exercise_intensity: approximate level of intensity (default 3) (int)
+    :param user_weight: weight of user (int)
+    :return: number of calories (float)
+    """
+    result = None
+    connection, crsr = psql_connection("database.ini", "postgresql")
```

Comment:
again, put connection to DB in a separate function as you already have duplicated code